client.js

```diff
@@ -796,7 +680,7 @@ async function instanceManagement(config, instance, instanceconfig) {
 		setInterval(getID, 10000);
 		getID();
 		function getID() {
-			messageInterface("/silent-command rcon.print(#game.connected_players)", function(playerCount) {
+			server.sendRcon("/silent-command rcon.print(#game.connected_players)").then((playerCount) => {
```

Comment:
But are we using only RCON going forward? stdin/stdout is a valid communication layer too and for some things probably is even better.